activity/activity.js

```diff
@@ -219,7 +219,7 @@ define(["webL10n",
         var alertNode = document.getElementById("activity-alert");
         if (alertNode && alertNode.parentNode) {
             alertNode.parentNode.removeChild(alertNode);
-        };
+        }
```

Comment:
... and yet change this differently?

Reply:
It has been the Formatter, I removed extra semicolons which were not needed